### ISSUE TITLE
fix(a11y/stepper): colour contrasts not accessible

### ISSUE THREAD

**Issue by Marine-Fabien** [adeo/mozaic-design-system#1266]
### I am opening an issue for

Component

### Package version

@mozaic-ds/styles@1.48.1

### Description

The colours contrasts of stepper are not fully accessible.

### Is this a regression?

- [ ] Yes
- [ ] No
- [ ] I don't know
